src/solver-service/lib/remote_commit.ml

```diff
@@ -0,0 +1,30 @@
+type t = {
+  repo : string;
+  hash : string
+} [@@deriving yojson]
```

Comment:
Can you add an mli file for this with doc comments about the module and what it is used for?